### ISSUE TITLE
feature: Add metadata so services can get correct information from the site

### ISSUE THREAD

**Issue by Alex-Cannon** [CodeForFoco/fd-servicedirectory#31]
It'd be nice to have metadata for services like linkedin to be able to pull the site picture, title, description, etc. so on sites like linkedin, the webpage will display an image.

This is done through the "og:image" "og:description", etc. meta tags and should be added the /public/index.html

**Comment by Alex-Cannon** [comment]
My pr did not work in production, so I'm re-opening this

**Comment by Alex-Cannon** [comment]
This is fixed locally, but not in production after #43 . I'm re-opening this issue.
